src/components/Search/Search.js

```diff
@@ -277,17 +282,25 @@ const Search = () => {
     // if user does not click on the same image, clear layers
     if (!currentImageClickedRef.current) {
       // only remove layers if the user clicked on a different footprint
-      if (clickedFootprintImageLayerRef.current)
+      if (clickedFootprintImageLayerRef.current) {
         clickedFootprintImageLayerRef.current.clearLayers()
-      if (clickedFootprintHighlightRef.current)
+      }
+      if (clickedFootprintHighlightRef.current) {
         clickedFootprintHighlightRef.current.clearLayers()
+      }
     }
 
     // remove existing footprints from map
     resultFootprintsRef.current.clearLayers()
   }
 
-  const getQueryVal = () => ({ 'eo:cloud_cover': { gte: 0, lte: _cloudCover } })
+  const getCloudCoverQueryVal = () => ({
+    'eo:cloud_cover': { gte: 0, lte: _cloudCover }
+  })
+
+  const getPolarizationQueryVal = () => ({
+    'sar:polarizations': { eq: ['VV', 'VH'] }
```

Comment:
now let's hope this works 🤞🏻

Reply:
unfortunately it doesn't, but `in` does

```suggestion
    'sar:polarizations': { 'in': ['VV', 'VH'] }
```

Reply:
This has been updated.